pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+    <groupId>io.dapr.spring</groupId>
+	<artifactId>spring-cloud-dapr-parent</artifactId>
+	<version>1.0.0-SNAPSHOT</version>
```

Comment:
Version should start with 0.0.1-SNAPSHOT. We will need to iterate on this before it can be called 1.0

Reply:
Modified version to 0.0.1-SNAPSHOT